samples/snippets/src/main/java/com/example/speech/SpeechModelAdaptationBeta.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.speech;
+
+// [START speech_transcribe_with_model_adaptation_gcs]
+import com.google.cloud.speech.v1p1beta1.AdaptationClient;
+import com.google.cloud.speech.v1p1beta1.CreateCustomClassRequest;
+import com.google.cloud.speech.v1p1beta1.CreatePhraseSetRequest;
+import com.google.cloud.speech.v1p1beta1.CustomClass;
+import com.google.cloud.speech.v1p1beta1.CustomClass.ClassItem;
+import com.google.cloud.speech.v1p1beta1.LocationName;
+import com.google.cloud.speech.v1p1beta1.PhraseSet;
+import com.google.cloud.speech.v1p1beta1.PhraseSet.Phrase;
+import com.google.cloud.speech.v1p1beta1.RecognitionAudio;
+import com.google.cloud.speech.v1p1beta1.RecognitionConfig;
+import com.google.cloud.speech.v1p1beta1.RecognitionConfig.AudioEncoding;
+import com.google.cloud.speech.v1p1beta1.RecognizeResponse;
+import com.google.cloud.speech.v1p1beta1.SpeechAdaptation;
+import com.google.cloud.speech.v1p1beta1.SpeechClient;
+import com.google.cloud.speech.v1p1beta1.SpeechRecognitionAlternative;
+import com.google.cloud.speech.v1p1beta1.SpeechRecognitionResult;
+import java.util.List;
+
+public class SpeechModelAdaptationBeta {
+
+  public void transcribeWithModelAdaptation() throws Exception {
```

Comment:
Can you update [this function signature to use main](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#method-structure)?

Reply:
Same elsewhere.